tasks/packager/tv-webos.js

```diff
@@ -101,11 +101,17 @@ function validateWebosVersion(version) {
 function confirmUseExistingData(userData, callback) {
     console.log('');
     console.log('      > [ Stored Information ]');
-    console.log('      > id          : ' + userData.id);
-    console.log('      > name        : ' + userData.name);
-    console.log('      > version     : ' + userData.version);
-    console.log('      > vendor      : ' + userData.vendor);
-    console.log('      > resolution  : ' + userData.resolution);
+    console.log('      > id                 : ' + userData.id);
+    console.log('      > name               : ' + userData.name);
+    console.log('      > version            : ' + userData.version);
+    console.log('      > vendor             : ' + userData.vendor);
+    console.log('      > icon               : ' + userData.icon);
+    console.log('      > largeIcon          : ' + userData.largeicon);
+    console.log('      > resolution         : ' + userData.resolution);
+    console.log('      > bgColor            : ' + userData.bgcolor);
```

Comment:
We decided to show stored information only for mandatory items we think.
If all kind of attributes are shown, log might be complex.

So, can you rollback this to have only id, name, version, vendor, resolution as previous?